slurm_predict.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -l
+#SBATCH --job-name=NeurWPredict
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=4
+#SBATCH --partition=normal
+#SBATCH --account=s83
+#SBATCH --output=lightning_logs/neurwp_pred_out.log
+#SBATCH --error=lightning_logs/neurwp_pred_err.log
+#SBATCH --time=03:00:00
+#SBATCH --no-requeue
+
+export PREPROCESS=false
+export NORMALIZE=false
+
```

Comment:
We still need these PREPROCESS steps because otherwise a fresh clone of the repo will be missing the graphs and receive such error `2: FileNotFoundError: [Errno 2] No such file or directory: 'graphs/multiscale/m2m_edge_index.pt'`

```suggestion

export PREPROCESS=true
export NORMALIZE=false

if [ "$PREPROCESS" = true ]; then
    echo "Create static features"
    srun -ul -N1 -n1 python create_static_features.py --boundaries 60
    echo "Creating mesh"
    srun -ul -N1 -n1 python create_mesh.py --dataset "cosmo" --plot 1
    echo "Creating grid features"
    srun -ul -N1 -n1 python create_grid_features.py --dataset "cosmo"
    if [ "$NORMALIZE" = true ]; then
        # This takes multiple hours!
        echo "Creating normalization weights"
        srun -ul -N1 -n1 python create_parameter_weights.py --dataset "cosmo" --batch_size 32 --n_workers 8 --step_length 1
    fi
fi
```